maintainers/maintainer-list.nix

```diff
@@ -6332,6 +6332,12 @@
     githubId = 209729;
     name = "Kevin Griffin";
   };
+  kevink = {
```

Comment:
This should be in a seperate commit

Reply:
Oh I'll try splitting it into two tomorrow, but this will mean this PR will be borked until the merge of the maintainer PR is through... It's kind of a catch 22

Reply:
You can do it in this PR, that's not an issue

Reply:
Ah sorry, I misread commit for PR, my bad

Reply:
No worries :)

Reply:
It's split now! :)